cpp/CMakeLists.txt

```diff
@@ -18,6 +18,37 @@
 cmake_minimum_required(VERSION 3.2)
 message(STATUS "Building using CMake version: ${CMAKE_VERSION}")
 
+cmake_policy(SET CMP0025 NEW)
+# IN_LIST if() operator
+cmake_policy(SET CMP0057 NEW)
```

Comment:
Could you add `if(POLICY CMP0057)` for CMake 3.2?
https://cmake.org/cmake/help/latest/policy/CMP0057.html says "This policy was introduced in CMake version 3.3.".